### PR TITLE
Add postgres-9.6 support

### PR DESCRIPTION
**Pull request by CyberDem0n**
starting from 9.6 we need to set wal_level = 'replica' which is alias for
'hot_standby'. It was working before without problems, but if somebody
change wal_level to 'replica', Patroni will expose pending_restart flag,
although restart in this case is not necessary.

Besides that bump versions of consul and etcd to the latest.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
+
+    ## Make sure we have a en_US.UTF-8 locale available
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
+
+    && apt-get update -y \
+    && apt-get install -y postgresql-contrib-${PGVERSION} \
+
+    # Remove the default cluster, which Debian stupidly starts right after installation of the packages
+    && pg_dropcluster --stop ${PGVERSION} main \
     && pip install python-etcd==0.4.3 python-consul==0.6.0 --upgrade \
```
Comment (alexeyklyukin) [inline Dockerfile]: There is python-consul==0.6.1 in pip

File: patroni/postgresql.py [hunk 2]
```diff
@@ -597,8 +599,10 @@ def start(self, block_callbacks=False):
         self._write_postgresql_conf()
         self.resolve_connection_addresses()
 
-        options = ' '.join("--{0}='{1}'".format(p, self._server_parameters[p]) for p, v in self.CMDLINE_OPTIONS.items()
-                           if self._major_version >= v[2])
+        opts = {p: self._server_parameters[p] for p, v in self.CMDLINE_OPTIONS.items() if self._major_version >= v[2]}
+        if self._major_version >= 9.6 and opts['wal_level'] == 'hot_standby':
+            opts['wal_level'] = 'logical'
```
Comment (alexeyklyukin) [inline patroni/postgresql.py]: shouldn't it be 'replica' instead of 'logical'?

Comment (CyberDem0n) [inline patroni/postgresql.py]: yep...

File: Dockerfile (hunk 1 above)
Comment (CyberDem0n) [inline Dockerfile]: It's not that important, but will fix it

**Comment by alexeyklyukin** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**